<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-220</y>
     <width>1011</width>
     <height>931</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>couch2.jpg</pixmap>
   </property>
  </widget>
  <widget class="QSlider" name="width_slider">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>430</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="sliderPosition">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="side_hight_slider">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>180</y>
     <width>21</width>
     <height>171</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="feet_height_slider">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>340</y>
     <width>21</width>
     <height>81</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="back_depth">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="back_height">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>21</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="side_width">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>130</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="back_cusion_height">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>110</y>
     <width>21</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
